XpringKit/ILP/DefaultIlpClient.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import SwiftGRPC
+
+/// Default client decorator for use in IlpClient
+public class DefaultIlpClient {
+
+    private let balanceNetworkClient: IlpNetworkBalanceClient
+    private let paymentNetworkClient: IlpNetworkPaymentClient
+
+    /// Initialize a new DefaultXpringClient
+    ///
+    /// - Parameters:
+    ///     - grpcUrl: The gRPC URL exposed by Hermes
```

Comment:
nit: The rest of this codebase uses `URL` instead of `Url` as do the built in Swift `URL` type. Can you fix the casing?